plugins/gssapi.c

```diff
@@ -1706,7 +2036,7 @@ static int gssapi_client_mech_step(void *conn_context,
 		
 		return SASL_INTERACT;
 	    }
-	}
+        }
```

Comment:
indentation change?

Reply:
Hmm.. Yeah, I saw that one at a glance when making the commit, but its a false positive, i think. 300 lines away... I'll double check.

Reply:
I really had to configure my Emacs in a weird way to get it to like the code and make indentation as similar as possible to what's already there... There was a TAB there before, somewhere along the way it got replaced by spaces. Fixed it so it's like before, anyway.